catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsApplication.java

```diff
@@ -215,6 +213,15 @@ public void init() {
           res.body(util.getJson(ImmutableMap.of(RESP_MSG, "Input was not valid.")));
         });
 
+    exception(
+        IllegalStateException.class,
+        (e, req, res) -> {
+          LOGGER.debug("Template input was not valid, {}", e.getMessage().toLowerCase());
```

Comment:
why log `e.getMessage().toLowerCase()` and not `e`?

Reply:
Great question. I don't remember. I think I worked under the assumption this was a user facing message. Reverting.